float64[] x
string action
---
int8 effect_id
float64[] y_predicted
